File: CS/Models/ListsDataHelper.cs
using System;
using System.Drawing;

public static class ListsDataHelper {
    public static string[] GetTeams() {
        return new string[] { "R&D", "Support", "Management" };
    }

    public static string[] GetLocations() {
        return new string[] { "Home", "Office", "Unknown" };
    }

    public static CustomerInfo GetSampleCustomer() {
        return new CustomerInfo() {
            Id = 0,
            FirstName = "John",
            BirthDate = new DateTime(1980, 1, 1),
            Notes = "Some text...",
            Age = 30,
            HireDate = new DateTime(2000, 1, 1),
            Salary = 20,
            PreferedColor = "red",
            Photo = (byte[])new ImageConverter().ConvertTo(SystemIcons.Application.ToBitmap(), typeof(byte[])),
            Teams = new string[] { "R&D" },
            Location = "Office",
            IsActive = true,
            PrimaryEmail = "test@test.com"
        };
    }
}

File: VB/Models/ListsDataHelper.vb
Imports System
Imports System.Drawing

Public NotInheritable Class ListsDataHelper

	Private Sub New()
	End Sub

	Public Shared Function GetTeams() As String()
		Return New String() { "R&D", "Support", "Management" }
	End Function

	Public Shared Function GetLocations() As String()
		Return New String() { "Home", "Office", "Unknown" }
	End Function

	Public Shared Function GetSampleCustomer() As CustomerInfo
		Return New CustomerInfo() With {.Id = 0, .FirstName = "John", .BirthDate = New Date(1980, 1, 1), .Notes = "Some text...", .Age = 30, .HireDate = New Date(2000, 1, 1), .Salary = 20, .PreferedColor = "red", .Photo = CType((New ImageConverter()).ConvertTo(SystemIcons.Application.ToBitmap(), GetType(Byte())), Byte()), .Teams = New String() { "R&D" }, .Location = "Office", .IsActive = True, .PrimaryEmail = "test@test.com"}
	End Function
End Class